VocaDbWeb/Scripts/Shared/WebLinkMatcher.ts

```diff
@@ -326,7 +345,99 @@ export default class WebLinkMatcher {
 			url: 'www.lagoa.jp/',
 			desc: 'THREE!',
 			cat: WebLinkCategory.Commercial,
-		} /* UtaiteDB */,
+		},
+		{
+			url: 'linkco.re/',
+			desc: 'TuneCore',
+			cat: WebLinkCategory.Official,
+		},
+		{
+			url: 's.awa.fm/',
+			desc: 'AWA',
+			cat: WebLinkCategory.Commercial,
+		},
+		{
+			url: 'recochoku.jp/',
+			desc: 'recochoku',
+			cat: WebLinkCategory.Commercial,
+		},
+		{
+			url: 'amazon.de/',
+			desc: 'Amazon',
+			cat: WebLinkCategory.Commercial,
+		},
+		{
+			url: 'kkbox.com/',
+			desc: 'KKBOX',
+			cat: WebLinkCategory.Commercial,
+		},
+		{
+			url: 'www.deezer.com/',
+			desc: 'Deezer',
+			cat: WebLinkCategory.Commercial
+		},
+		{
+			url: 'seiga.nicovideo.jp/',
+			desc: 'NicoNicoSeiga',
+			cat: WebLinkCategory.Official
+		},
+		{
+			url: 'commons.nicovideo.jp/',
+			desc: 'Nicommons',
+			cat: WebLinkCategory.Official
+		},
+		{ 
+			url: 'linktr.ee/',
+			desc: 'Linktree',
+			cat: WebLinkCategory.Reference
+		},
+		{ 
+			url: 'music.dmkt-sp.jp/',
+			desc: 'd music',
+			cat: WebLinkCategory.Commercial 
+		},
+		{ 
+			url: 'music.hikaritv.net/',
+			desc: 'Hikari TV Music',
+			cat: WebLinkCategory.Commercial 
+		},
+		{ 
+			url: 'musicstore.auone.jp/',
+			desc: 'Music Store',
+			cat: WebLinkCategory.Commercial
+		},
+		{
+			url: 'nex-tone.link/',
+			desc: 'NexTone.Link',
+			cat: WebLinkCategory.Reference
+		},
+		{
+			url: 'pc.animelo.jp/',
+			desc: 'animelo mix',
+			cat: WebLinkCategory.Commercial
+		},
+		{
+			url: 'pc.dwango.jp/',
+			desc: 'dwango.jp',
+			cat: WebLinkCategory.Commercial
+		},
+		{
+			url: 'uta.573.jp/',
+			desc: 'KONAMI MUSIC',
+			cat: WebLinkCategory.Commercial
+		},
+		{
+			url: 'www.beatport.com/',
+			desc: 'Beatport',
+			cat: WebLinkCategory.Commercial
+		},
+		{
+			url: 'www.e-onkyo.com/',
+			desc: 'e-onkyo music',
+			cat: WebLinkCategory.Commercial
+		},
+		
+		/* UtaiteDB */,

```

Comment:
Please drop the redundant empty line and comma, which result in a compile error. I recommend you run the `npm run watch` (or `npm run dev`) command before creating a pull request, because it can catch errors (and warnings) at compile time rather than at run-time. (or you could set up a command like `npm run lint` or `npm run eslint` if you want.) See the wiki for more detailed information.